tm2/pkg/std/memfile.go

```diff
@@ -38,9 +38,16 @@ func (mempkg *MemPackage) IsEmpty() bool {
 }
 
 var (
-	rePkgName      = regexp.MustCompile(`^[a-z][a-z0-9_]*$`)
-	rePkgOrRlmPath = regexp.MustCompile(`^gno\.land\/(?:p|r)(?:\/_?[a-z]+[a-z0-9_]*)+$`)
-	reFileName     = regexp.MustCompile(`^([a-zA-Z0-9_]*\.[a-z0-9_\.]*|LICENSE|README)$`)
+	rePkgName = regexp.MustCompile(`^[a-z][a-z0-9_]*$`)
+	/*
+		in case we use RE2 external lib (e.g. dlclark/regexp2), we can do this
+		^(?=.{0,256}$)gno\.land\/(?:p|r)(?:\/_?[a-z]+[a-z0-9_]*)+$
+		with lookahead operation to limit the length
+		but for now we just check it with rePkgOrRlmLenLimit.
+	*/
+	rePkgOrRlmPath     = regexp.MustCompile(`^gno\.land\/(?:p|r)(?:\/_?[a-z]+[a-z0-9_]*)+$`)
+	reFileName         = regexp.MustCompile(`^([a-zA-Z0-9_]*\.[a-z0-9_\.]*|LICENSE|README)$`)
+	rePkgOrRlmLenLimit = 256
```

Comment:
Usually regex expressions start with `re`. What do you think about naming this `pathLengthLimit`, and making it a `const`?

Reply:
This should be `const`, yes.

Reply:
I changed it sir @zivkovicmilos